Dockerfile

```diff
@@ -3,6 +3,18 @@ WORKDIR /go/src/github.com/openshift/cluster-node-tuning-operator
 COPY . .
 RUN make build
 
+FROM centos:7 as tuned
+WORKDIR /root
+COPY assets/tuned /root
+RUN INSTALL_PKGS=" \
+      git rpm-build make desktop-file-utils patch \
+      " && \
+    yum install --setopt=tsflags=nodocs -y $INSTALL_PKGS && \
+    cd daemon && \
+    LC_COLLATE=C cat ../patches/*.diff | patch -Np1 && \
+    make rpm PYTHON=/usr/bin/python && \
+    rm -rf /root/rpmbuild/RPMS/noarch/tuned-profiles-nfv*
```

Comment:
what are the tuned-profiles-nfv* ?

Reply:
Thanks for looking!
[Those packages](https://github.com/redhat-performance/tuned/blob/master/doc/manual/modules/performance/ref_tuned-profiles-distributed-with-rhel.adoc) carry `realtime-virtual-host` and `realtime-virtual-guest` profiles.
```
Wrote: /root/rpmbuild/RPMS/noarch/tuned-profiles-nfv-guest-2.14.0-1.el7.noarch.rpm
Wrote: /root/rpmbuild/RPMS/noarch/tuned-profiles-nfv-host-2.14.0-1.el7.noarch.rpm
Wrote: /root/rpmbuild/RPMS/noarch/tuned-profiles-nfv-2.14.0-1.el7.noarch.rpm
```
To-date, NTO shipped only the "realtime" profile.  Nothing with this PR changes that.  The realtime profile is still being shipped and on top of that many others that were not shipped.  `realtime-virtual-host` and `realtime-virtual-guest` profiles are the only upstream profiles which this PR excludes.  The main reasons for this decision is that they are not written in a container-compatible way and packaging issues (dependency on an external qemu-kvm-tools-rhev).  Adjusting and including these profiles will be a subject of future work.

Reply:
I'm in fact contemplating a removal of a few more profiles and tools to keep the image smaller and shipping only profiles that are known to work 100% with NTO.